R/compareScenarios.R

```diff
@@ -36,6 +36,8 @@ compareScenarios <- function(mif, hist,
                              y_bar=c(2010,2030,2050,2100),
                              reg=NULL, mainReg="GLO", fileName="CompareScenarios.pdf",
                              sr15marker_RCP=NULL) {
+  message("compareScenarios is outdated and may break - use compareScenarios2 instead!")
```

Comment:
@chroetz: here I added the warning message

Reply:
Thanks! That is helpful. 
I think, the function should be removed eventually.
Maybe it can even be removed now as it doesn't work on current mifs?